src/components/organisms/RadioModal/RadioModal.tsx

```diff
@@ -46,9 +46,11 @@ export const RadioModal: React.FunctionComponent<RadioModalPropsType> = ({
   );
 
   return (
-    <div className="radio-modal-container">
+    <div role="dialog" className="radio-modal-container">
       {!isRadioPlaying ? (
-        <Button onClick={handleEnableButtonClick}>Enable radio</Button>
+        <Button aria-label="Enable radio" onClick={handleEnableButtonClick}>
+          Enable radio
+        </Button>
```

Comment:
Do we need to use an `aria-label` when the button already wraps it's text? I would assume that is fine as is from an a11y standpoint.

Reply:
![image](https://user-images.githubusercontent.com/84011566/125524245-6b83d353-3dbe-4a46-912a-962bc2a39841.png)

Reply:
That’s talking about an entirely different button though, and that chat button has no text on it because it’s just an image icon. So that makes sense. But in this case the button already seems to have text within it. So we shouldn’t have any need to duplicate that text again in the aria-label

Reply:
@oleksii-front ^?

Reply:
@mike-lvov @0xdevalias  agree.